<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managebooks</class>
 <widget class="QWidget" name="managebooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget_manage_bok">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_edit_book_2">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="edit_account_title_2">
           <property name="styleSheet">
            <string notr="true">QLabel#edit_account_title {
	height: 35pt;
    border-style: solid;
	border-width: 0px;
    border-color: rgb(255,255,255);
	border-top-color: rgb(204,239,254);
	font:normal 200 14pt &quot;Myriad Pro&quot;;
	min-height: 75px;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;------ EDIT BOOK ------&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLabel" name="label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLabel" name="notification_author_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="notification_bookname_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="notification_publishedday_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLabel" name="notification_kind_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_notification_of_browse_pdf">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Included contents!</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_content_2">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="19" column="1">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_publisheddate_2"/>
           </item>
           <item row="10" column="1">
            <widget class="QLineEdit" name="lineEdit_genre_2"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_bookname_2"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_published_date_2">
             <property name="text">
              <string>Published Date*: </string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_author_2"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_genre_2">
             <property name="text">
              <string>Genre*: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_bookid">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_publisher_2">
             <property name="text">
              <string>Publisher:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_book_2">
             <property name="text">
              <string>Book ID*: </string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLineEdit" name="lineEdit_amount_2"/>
           </item>
           <item row="16" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_show_image">
               <property name="minimumSize">
                <size>
                 <width>151</width>
                 <height>151</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>151</width>
                 <height>151</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="14" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Content (PDF): </string>
             </property>
            </widget>
           </item>
           <item row="17" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="pushButton_browse_image_2">
             <property name="minimumSize">
              <size>
               <width>161</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>161</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_book_name_2">
             <property name="text">
              <string>Book's name*: </string>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Image*: </string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Amount of books: </string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_author_2">
             <property name="text">
              <string>Author*: </string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="lineEdit_publisher_2"/>
           </item>
           <item row="18" column="1">
            <widget class="QLabel" name="notification_show_image_2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="15" column="1">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_OK_2">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Cancel_2">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_add_book">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="edit_account_title_3">
           <property name="styleSheet">
            <string notr="true">QLabel#edit_account_title {
	height: 35pt;
    border-style: solid;
	border-width: 0px;
    border-color: rgb(255,255,255);
	border-top-color: rgb(204,239,254);
	font:normal 200 14pt &quot;Myriad Pro&quot;;
	min-height: 75px;
}</string>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;------ ADD BOOK ------&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="13" column="2">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="15" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QLabel" name="notification_author_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="notification_publisheddate_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="notification_bookname3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="lineEdit_author_3"/>
           </item>
           <item row="12" column="2">
            <widget class="QLineEdit" name="lineEdit_amount_3"/>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_book_3">
             <property name="text">
              <string>Book ID*: </string>
             </property>
            </widget>
           </item>
           <item row="16" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_show_image_3">
               <property name="minimumSize">
                <size>
                 <width>151</width>
                 <height>151</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>151</width>
                 <height>151</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_book_name_3">
             <property name="text">
              <string>Book's name*: </string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Amount of books: </string>
             </property>
            </widget>
           </item>
           <item row="16" column="1">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Image*: </string>
             </property>
            </widget>
           </item>
           <item row="18" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="pushButton_browse_image_3">
             <property name="minimumSize">
              <size>
               <width>161</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>161</width>
               <height>23</height>
              </size>
             </property>
             <property name="text">
              <string>Browse</string>
             </property>
            </widget>
           </item>
           <item row="14" column="1">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Content (PDF): </string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="lineEdit_publisher_3"/>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_author_3">
             <property name="text">
              <string>Author*: </string>
             </property>
            </widget>
           </item>
           <item row="14" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_notification_of_browse_pdf_3">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Not include any contents!</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_browse_content_3">
               <property name="text">
                <string>Browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="2">
            <widget class="QLineEdit" name="lineEdit_publisheddate_3"/>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="label_genre_3">
             <property name="text">
              <string>Genre*: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_bookid_3">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="label_publisher_3">
             <property name="text">
              <string>Publisher:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_published_date_3">
             <property name="text">
              <string>Published Date*: </string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QLineEdit" name="lineEdit_genre_3"/>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_bookname_3"/>
           </item>
           <item row="16" column="0">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QLabel" name="notification_kind_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="19" column="2">
            <widget class="QLabel" name="notification_show_image_3">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="pushButton_OK_3">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_Cancel_3">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
